src/components/Integrations/BigQuery/FreeAccount/FreeAccount.tsx

```diff
@@ -11,6 +12,8 @@ const FreeAccount: React.SFC<IFreeAccountProps> = (props) => {
     const intl = useIntl();
     const _ = (id: string, values: any) => intl.formatMessage({ id: id }, values);
 
+    const [redirect, setRedirect] = useState(false);
+
     const tittle = (
```

Comment:
no need to use <div> as <Fragment>. You could use <>

Reply:
same case for 'list'

Reply:
in this case the hook was removed because button was replaced with <a> tag